core/src/scss/components/main-nav/_main-nav--dark.scss

```
@@ -87,31 +85,31 @@
         &,
         &.su-main-nav__item--expanded {
           > a::after {
-            background: url("#{$su-image-path}/caret-down-white.svg") no-repeat 0 0;
+            background: url("#{$image-path}/caret-down-white.svg") no-repeat 0 0;
```

```suggestion
            background: url("#{$su-image-path}/caret-down-white.svg") no-repeat 0 0;
```
I think this one needs to keep the `su-` prefix.